.circleci/config.yml

```diff
@@ -585,48 +494,12 @@ commands:
             wget -qO- https://raw.githubusercontent.com/nvm-sh/nvm/v0.34.0/install.sh | bash
             bash -i -c 'nvm install'
   prepare_accessibility_test:
-    parameters:
-      db_dump:
-        type: string
-        default: "db_dump.sql"
     steps:
-      - browser-tools/install-chrome
-      - browser-tools/install-chromedriver
-      - restore_cache:
-          key: dep-cache-{{ .Environment.CACHE_VERSION }}-{{ checksum "./package-lock.json" }}
-      - run:
-          name: Install dependencies
-          command: bash -i -c 'npm ci'
```

Comment:
We were installing the dependencies without checking the cache first. Fixed it so dependencies are only installed if the cache doesn't have them, which saves us about 54 seconds